builtins/amp-img.js

```diff
@@ -64,17 +64,12 @@ export class AmpImg extends BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
-    return true;
-  }
-
-  /** @override */
   isRelayoutNeeded() {
     return true;
   }
 
   /** @override */
-  layoutCallback() {
+  prerenderCallback() {
```

Comment:
This could work, but worth discussing and checking. The issue here is that for the image prerender == layout essentially. This actually may not always be the case. For instance, we may later introduce levels of prerender where we'd prerender image with a much lower quality image and will upgrade to full quality when it's scrolled into view. 

But let's assume for now, prerender == layout. This means that runtime will mark the resource as "prerendered", but not yet "laid out". At a different time, it will call layout on it again. At that point the layout should be almost instantaneous. This doesn't sound too bad for an image, but I wonder if calling something "prerendered" when it was actually "laid out" could cause us any trouble.

No action item here - but let's discuss.